arraycontext/fake_numpy.py

```diff
@@ -128,48 +127,8 @@ def _get_fake_numpy_linalg_namespace(self):
 
         # FIXME:
         # "interp",
-
         })
 
-    _numpy_to_c_arc_functions = {
-            "arcsin": "asin",
-            "arccos": "acos",
-            "arctan": "atan",
-            "arctan2": "atan2",
-
-            "arcsinh": "asinh",
-            "arccosh": "acosh",
-            "arctanh": "atanh",
-            }
-
-    _c_to_numpy_arc_functions = {c_name: numpy_name
-            for numpy_name, c_name in _numpy_to_c_arc_functions.items()}
-
-    def __getattr__(self, name):
-        def loopy_implemented_elwise_func(*args):
-            actx = self._array_context
-            prg = _get_scalar_func_loopy_program(actx,
```

Comment:
It looks like you meant to move `_get_scalar_func_loopy_program`, but there's still a copy of that (now-unused) function hanging out at the top of this file.

Reply:
Ohh got it. Thanks for the feedback